src/sentry/search/utils.py

```diff
@@ -277,7 +277,9 @@ def get_latest_release(projects, environments, organization_id=None):
     )
 
 
-def parse_release(value, projects, environments, organization_id=None):
+def parse_release(
```

Comment:
Nit: I would split this out as part of a different PR since it is not really related

Reply:
Tbh I'm not sure why I did this at all, wondering if my IDE did it, since marking these all as `object` isn't helpful